active_directory/ci/active_directory.rake

```diff
@@ -0,0 +1,65 @@
+require 'ci/common'
```

Comment:
To be consistent with other integrations, this should be shipped in `active_directory/test_ci/active_directory.rake`